src/setup/rgw.py

```diff
@@ -478,3 +481,22 @@ def _generate_ssl_cert(conf: MappedConf):
                     cert_path=ssl_cert_path, dns_list=SSL_DNS_LIST, **ssl_cert_configs)
             except SSLCertificateError as e:
                 raise SetupError(errno.EINVAL, f'Failed to generate self signed ssl certificate: {e}')
+
+    def _logrotate_generic(conf):
+        """ Configure logrotate utility for rgw logs."""
+        log_dir = conf.get(LOG_PATH_KEY)
+        log_file_path = os.path.join(log_dir, COMPONENT_NAME, Rgw._machine_id)
+        # rename ceph logrotate file to component's name logrotate.
+        # For eg:
+        # '/etc/logrotate.d/ceph' -> '/etc/logrotate.d/cortx_<component_name>'
```

Comment:
process is still radosgw, we need to keep any rgw related code or config as generic as possible. so here filename should also be radosgw

Reply:
Done